### PR TITLE
[ELY-2439] Add a test class for RegexNameValidatingRewriter

### PR DESCRIPTION
**Pull request by lvydra**
https://issues.redhat.com/browse/ELY-2439

### CODE REVIEW

File: tests/base/src/test/java/org/wildfly/security/auth/server/RegexNameRewriterTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import org.wildfly.security.auth.realm.FileSystemSecurityRealm;
+import org.wildfly.security.auth.util.RegexNameRewriter;
+
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class RegexNameRewriterTest {
```
Comment (skyllarr) [inline tests/base/src/test/java/org/wildfly/security/auth/server/RegexNameRewriterTest.java]: @lvydra Thank you for the PR! 

This specific issue is to add tests for https://github.com/wildfly-security/wildfly-elytron/blob/1.x/auth/util/src/main/java/org/wildfly/security/auth/util/RegexNameValidatingRewriter.java specifically. This commit adds tests for RegexNameRewriter which can be part of issue [ELY-2446](https://issues.redhat.com/browse/ELY-2446). 

You can add new commit with issue ELY-2446 and add new tests for RegexNameRewriter under it. But for ELY-2439 we are looking to add tests for RegexName**Validating**Rewriter.

Comment (lvydra) [reply tests/base/src/test/java/org/wildfly/security/auth/server/RegexNameRewriterTest.java]: @Skyllarr Thank you, I misread the description.

**Comment by lvydra** [comment]
@Skyllarr Fixed, the test should be now for the right rewriter :-)

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
@lvydra Thank you!

**Review by fjuma — APPROVED** [review]
Thanks @lvydra!

**Pull request merged**